common/test.c

```diff
@@ -501,8 +503,11 @@ p11_test_copy_setgid (const char *input)
 		return NULL;
 	}
 
-	path = strdup ("/tmp/test-setgid.XXXXXX");
-	assert (path != NULL);
+	tmpdir = getenv("TMPDIR");
+	if (tmpdir == NULL)
+		tmpdir = "/tmp";
+
+	assert(asprintf(&path, "%s/test-setgid.XXXXXX", tmpdir) >= 0);
```

Comment:
I've added the space, however I am not sure about the second point. I see that the test suite is built around assert(), and that construction in most of the tests. Does it make sense to avoid assert on this particular test?